pkg/infradriver/ssh_infradriver.go

```diff
@@ -337,11 +338,11 @@ func (d *SSHInfraDriver) GetHostsPlatform(hosts []net.IP) (map[v1.Platform][]net
 }
 
 func (d *SSHInfraDriver) GetClusterRootfsPath() string {
-	return common.DefaultTheClusterRootfsDir(d.clusterName)
+	return filepath.Join(common.DefaultSealerDataDir, d.clusterName, "rootfs")
```

Comment:
why not keep it?

Reply:
here i thought, the only way we get the cluster value such as "GetClusterRootfsPath" and the same kind parameters. not from common pkg directly .

Reply:
And then, should we delete the function `common.DefaultTheClusterRootfsDir`?